### PR TITLE
DOC-778 What's new 4.40.0 and 4.41.0

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves [DOC-778](https://redpandadata.atlassian.net/browse/DOC-778)
Review deadline: 2nd December

## Page previews

[What's new](https://deploy-preview-112--redpanda-connect.netlify.app/redpanda-connect/get-started/whats-new/)

**Links to licensing page will be broken until related docs are released**

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

[DOC-778]: https://redpandadata.atlassian.net/browse/DOC-778?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Jeffail — APPROVED** [review]

**Review by Feediver1 — APPROVED** [review]
lgtm

File: modules/get-started/pages/whats-new.adoc [hunk 1]
```diff
@@ -6,6 +6,51 @@ This topic includes new content added from version 4.29.0 onwards.
...
+* Options to optimize the build of the data output from the xref:components:outputs/snowflake_streaming.adoc[`snowflake_streaming` output].
+
+Other changes:
+
+* The following components now use the https://github.com/twmb/franz-go/tree/master/pkg/sr[Franz Kafka Schema Registry client library^]: xref:components:inputs/schema_registry.adoc[`schema_registry` input] and xref:components:outputs/schema_registry.adoc[output], and the xref:components:processors/schema_registry_encode.adoc[`schema_registry_encode`] and xref:components:processors/schema_registry_decode.adoc[`schema_registry_decode`] processors.
+* The xref:components:outputs/snowflake_streaming.adoc[`snowflake_streaming` output] can now automatically create tables in Snowflake, when the `schema_evolution.enabled` field is set to `true`.
+
+==== Deprecations:
+
+This release removes:
```
Comment (JakeSCahill) [inline modules/get-started/pages/whats-new.adoc:49]: It'd be useful to make it easy to scan for affected components and the new name. Alos, we should call out any actions users must take. Maybe something like:

Inputs

**Deprecated field**: `kafka_timestamp`
**Affected components**:
- redpanda`
- ...
**Replaced by**: `kafka_timestamp_ms`

Action required

Modify all systems that process these fields to ensure compatibility with the new field names.

File: modules/get-started/pages/whats-new.adoc [hunk 2]
```diff
@@ -6,6 +6,51 @@ This topic includes new content added from version 4.29.0 onwards.
 
 For a full list of product updates, and details of updates prior to version 4.29.0, see the https://github.com/redpanda-data/connect/releases[Redpanda Connect release notes^]. For new Cloud-specific features, see xref:redpanda-cloud:get-started:whats-new-cloud.adoc[].
 
+== Version 4.42.0
+
+This release introduces a xref:guides:bloblang/playground.adoc[Bloblang playground] where you can try out your Bloblang syntax and get immediate feedback. It also updates Redpanda Connect licensing checks for Self-Managed. Starting from version 4.42.0, you must xref:get-started:licensing.adoc[apply your Enterprise Edition license] to use any Enterprise connectors.
```
Comment (JakeSCahill) [inline modules/get-started/pages/whats-new.adoc:11]:
```suggestion
This release introduces a xref:guides:bloblang/playground.adoc[Bloblang playground], where you can try out your Bloblang syntax and get immediate feedback. It also adds licensing checks for Redpanda Connect. Starting from version 4.42.0, you must xref:get-started:licensing.adoc[add your Enterprise Edition license] to Redpanda Connect before using any Enterprise connectors.
```

File: modules/get-started/pages/whats-new.adoc [hunk 3]
```diff
@@ -6,6 +6,51 @@ This topic includes new content added from version 4.29.0 onwards.
...
+
+To create your own CGO builds, run the following command:
+
+```bash
+CGO_ENABLED=1 make TAGS=x_benthos_extra redpanda-connect. 
+```
+
+== Version 4.40.0
+
+This update includes a beta release of the xref:components:inputs/pg_stream.adoc[`pg_stream` input] for Self-Managed and Cloud, which captures data changes made to a PostgreSQL database in real time and streams them to Redpanda Connect.
```
Comment (JakeSCahill) [inline modules/get-started/pages/whats-new.adoc:29]:
```suggestion
This update includes a beta release of the xref:components:inputs/pg_stream.adoc[`pg_stream` input] for Redpanda Connect Self-Managed and Cloud, which captures data changes made to a PostgreSQL database in real-time and streams them to Redpanda Connect.
```

File: modules/get-started/pages/whats-new.adoc [hunk 4]
```diff
@@ -6,6 +6,51 @@ This topic includes new content added from version 4.29.0 onwards.
...
+
+=== Component updates
+
+This release adds:
+
+* The `metadata_max_age` field to the xref:components:outputs/redpanda_migrator_offsets.adoc[`redpanda_migrator_offsets` output].
```
Comment (JakeSCahill) [inline modules/get-started/pages/whats-new.adoc:35]: Maybe put this in a table to make it easy to scan.

**Review by JakeSCahill — APPROVED** [review]
Looks good. left a few non-blocking suggestions.

**Review by Jeffail — APPROVED** [review]

File: modules/get-started/pages/whats-new.adoc (hunk 3 above)
Comment (asimms41) [reply modules/get-started/pages/whats-new.adoc:29]: Updated.

File: modules/get-started/pages/whats-new.adoc (hunk 4 above)
Comment (asimms41) [reply modules/get-started/pages/whats-new.adoc:35]: I'll open a ticket for this, so I can go back over all the other releases and make them consistent.

**Review by Jeffail — APPROVED** [review]

**Pull request merged**